coralproject/talk
# [CORL-1252] Live Viewer Counts

## What does this PR do?

Adds experimental live viewer counts to the stream and stories pages. This requires the `VIEWER_COUNT` feature flag.



## What changes to the GraphQL/Database Schema does this PR introduce?

- Add `Story.viewerCount`
- Add `FEATURE_FLAG.VIEWER_COUNT`



## How do I test this PR?

Enable the `VIEWER_COUNT` feature flag, navigate to the stream and stories pages and see the changes!
